rave-test/src/main/java/com/uber/rave/IntCreator.java

```diff
@@ -27,16 +27,16 @@
  * This class is creates valid and invalid permutations of a long argument based on the annotation
  * restrictions.
  */
-public final class LongCreator extends ObjectCreator<Long> {
```

Comment:
Shouldn't you keep this for LongDef support in the future? Let's create a new `IntCreator` rather than introduce churn here. Also the docs on this were not updated anyway

Reply:
I was thinking we'd introduce LongCreator as part of LongDef support. Introducing a new `IntCreator` requires migrating old call sites since we've made a breaking change in the constructor. 

Updated the doc at the top of this class in `28172a3`. Let me know if that's cool :)